index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title></title>
+    <link rel="stylesheet" href="style.css">
+    <header>
+      <h1>Welcome to Salmon Cookies!</h1>
+    </header>
+    <img src="https://raw.githubusercontent.com/codefellows/seattle-201d19/master/class-06-dom-objects-domain_modeling/assets/salmon.png" />
+  </head>
+  <body>
+    <p id="main-text">Welcome to the greatest invention since sliced bread! Here at Salmon Cookies, we offer only the best cookies made of salmon! They're in the shape of salmon, they have the fun of salmon, they just don't taste like salmon! Thanks to being ground up super fine, you can't taste the salmon. It's the best pairing for coffee on an overcast day in Seattle!</p>
+    <h3>We serve:</h3>
+    <ul>
+      <li>Coffee!</li>
+      <li>Salmon Cookies!</li>
+      <li>Inspiration!</li>
+      <li>Happiness in baked bread form!</li>
+    </ul>
+    <h3>We are located at:</h3>
+    <ul>
+      <li>1st and Pike</li>
+      <li>Seatac Airport</li>
+      <li>Seattle Center</li>
+      <li>Capitol Hill</li>
+      <li>Alki</li>
+    </ul>
+    <p id="inspiration">Remember! Life might get hard, but always swim upstream!</p>
+    <script src="app.js" type="text/javascript"></script>
```

Comment:
Linking to app.js on this page will cause an error, because the id's your code is looking for don't exist on this page.